rules-reviewed/eap7/eap6/eap6.windup.xml

```diff
@@ -21,7 +21,7 @@
     <rules>
         <rule id="eap6-01000">
             <when>
-                <javaclass references="{type}">
+                <javaclass references="javax.ejb.EntityBean">
```

Comment:
This is fine, but the text below still mentions CMP, while it is about _all_ entity beans.